about/contributing_to_mpf_docs.rst

```diff
@@ -87,11 +87,11 @@ mpf-docs repo so you can work on it offline, do the following:
 #. Switch to the branch corresponding to the version of the docs you want to
    work with (usually *dev* or *latest*).
 #. Makes your changes.
-#. Add your name to the ``/authors/index.rst`` doc.
+#. Add your name to the ``/about/authors.rst`` doc.
```

Comment:
The `authors.rst` file seemed to be correct file since that has the authors in it.